creds.env.example

```diff
@@ -1,10 +0,0 @@
-# nautobot_chatops
```

Comment:
Why delete this file?

Reply:
It is in the wrong location. The real file is in `development/creds.env.example`